### DISCUSSION TITLE
Add 'go to top' button on Readme.md

### DISCUSSION THREAD

**Discussion by jxswxnth** [community/community#42712] [Accessibility] [answered]
For long Readme pages, it's better to have a 'go to top' button as an arrow provided which navigates back to top of the page.

**Comment by subidit** [comment]
Since you can't use `javascript` in markdown files, hence implementing a go to top **button** is not possible. However there is a work-around that is commonly used to get this exact  same result. All you need to do is create a link by creating a section (such as # H1) and then link to that section with `go to top` link. 

For example- if you have
# Section 1
then at the bottom of your page you could have
 [Go to top](#section-1)

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @jxswxnth 

You can create a link in markdown and point to some text, as shown below.

If you want, you can look at my repository and try to do [https://github.com/SilasRodrigues19/Portfolio/blob/main/README.md](https://github.com/SilasRodrigues19/Portfolio/blob/main/README.md)

![image](https://user-images.githubusercontent.com/47532703/209715457-cc10cbe8-68b3-42f1-9939-72ec48a3694a.png)
![image](https://user-images.githubusercontent.com/47532703/209715487-52fd8be5-b71f-4809-bfdf-aeb9c1441d08.png)
![chrome_bTevDqTz6i](https://user-images.githubusercontent.com/47532703/209715571-f1075c0c-a017-4c45-8057-ca9362ed24a3.gif)

**Comment by cehfisher** [comment]
Hi @jxswxnth - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/42712#discussioncomment-4504407). Could you please close this issue? Thank you! (cc @queenofcorgis)
